docs/connect/java/postgresql-jdbc.md

```diff
@@ -0,0 +1,136 @@
+(pgjdbc)=
+(postgresql-jdbc)=
+
+# PostgreSQL JDBC
+
+:::{include} /_include/links.md
+:::
+
+:::{div} sd-text-muted
+Connect to CrateDB using PostgreSQL JDBC.
+:::
+
+:::{rubric} About
+:::
+
+:::{div}
+The [PostgreSQL JDBC Driver] is an open-source JDBC driver written in
+Pure Java (Type 4), which communicates using the PostgreSQL native
+network protocol.
+:::
+
+:::{rubric} Synopsis
+:::
+
+`example.java`
+```java
+import java.sql.*;
+
+void main() throws SQLException {
+
+    // Connect to database.
+    Properties properties = new Properties();
+    properties.put("user", "crate");
+    properties.put("password", "crate");
+    Connection conn = DriverManager.getConnection(
+        "jdbc:postgresql://localhost:5432/doc?sslmode=disable",
+        properties
+    );
+    conn.setAutoCommit(true);
+
+    // Invoke query.
+    Statement st = conn.createStatement();
+    st.execute("SELECT mountain, height FROM sys.summits ORDER BY height DESC LIMIT 5;");
+
+    // Display results.
+    ResultSet rs = st.getResultSet();
+    while (rs.next()) {
+        System.out.printf(Locale.ENGLISH, "%s: %d\n", rs.getString(1), rs.getInt(2));
+    }
+    conn.close();
+
+}
+```
+
+:::{include} ../_cratedb.md
+:::
+Download JAR file.
+```shell
+wget https://repo1.maven.org/maven2/org/postgresql/postgresql/42.7.8/postgresql-42.7.8.jar
+```
+:::{dropdown} Instructions for Windows users
+If you don't have the `wget` program installed, for example on Windows, just
+download the JAR file using your web browser of choice.
+If you want to use PowerShell, invoke the `Invoke-WebRequest` command instead
+of `wget`.
+```powershell
+Invoke-WebRequest https://repo1.maven.org/maven2/org/postgresql/postgresql/42.7.8/postgresql-42.7.8.jar -OutFile postgresql-42.7.8.jar
+```
+:::
+Invoke program. Needs Java >= 25 ([JEP 330]).
```

Comment:
PostgreSQL-JDBC works with Java >= 1.8, and the given command does not need JEP 330 afaik.

```suggestion
Invoke program. Needs Java >= 1.8.
```